client/src/components/ActionColumn.js

```diff
@@ -33,6 +33,18 @@ const ActionColumn = props => {
     return (
       <Table.Cell>
         <View id={id} />
+        {status === "Waiting" ? (
```

Comment:
I think this condition is not enough, as the delete button should be displayed only for users and not interpreters.

Reply:
line 32 there is a condition userRole=== User